isotovideo

```diff
@@ -137,28 +199,35 @@ eval {
 
     $runner->handle_commands;
 
-    $return_code = 0;
+    $RETURN_CODE = EXIT_STATUS_OK;
 
     # enter the main loop: process messages from autotest, command server and backend
     $runner->run;
 
+    if ($options{'exit-status-from-test-results'}) {
+        # Ideally: $RETURN_CODE = $runner->command_handler->test_completed == 0;
+        #          should do it, but autotest returns success if the last
+        #          module ran successfully. Rendering this useless...
```

Comment:
I don't think even "ideally" that "test_completed" should yield a return_code. That method can just tell if all modules had been executed, regardless of their individual result. So I guess you should just ditch this comment

Reply:
it's not a return code, but a count of tests executed. But it doesn't behave as I'd expected. In any case it's deleted now.